### PR TITLE
Add build_reqProc_x86_64_Ubuntu job

### PR DESCRIPTION
**Pull request by shrivara**
https://github.com/Shippable/beta/issues/773

### CODE REVIEW

File: shippable.jobs.yml [hunk 1]
```diff
@@ -219,14 +219,30 @@ jobs:
...
+        - script: ./IN/reqProc_repo/gitRepo/build.sh drydock x86_64 Ubuntu
+
   - name: package_reqExec_x86_64_Ubuntu
     type: runSh
     steps:
       - IN: config_repo
         switch: off
       - IN: micro_repo
+        switch: off
       - IN: reqExec_repo
```
Comment (shrivara) [inline shippable.jobs.yml]: This dependency was missed. We have this for all of our build job which depend on the microbase image.

File: shippable.jobs.yml [hunk 2]
```diff
@@ -219,14 +219,30 @@ jobs:
...
+      - TASK:
+        - script: ./IN/reqProc_repo/gitRepo/build.sh drydock x86_64 Ubuntu
+
   - name: package_reqExec_x86_64_Ubuntu
     type: runSh
     steps:
       - IN: config_repo
         switch: off
       - IN: micro_repo
+        switch: off
```
Comment (shrivara) [inline shippable.jobs.yml]: Switching this off as this is only needed when reports change, which isn't all that often.

File: shippable.jobs.yml [hunk 3]
```diff
@@ -219,14 +219,30 @@ jobs:
...
       - IN: reqExec_repo
       - IN: aws_v2_bits_access_cli
+      - IN: microbase_img
```
Comment (shrivara) [inline shippable.jobs.yml]: This dependency was missed. We have this for all of our build job which depend on the microbase image.

File: shippable.jobs.yml [hunk 4]
```diff
@@ -219,14 +219,30 @@ jobs:
...
+      - IN: package_reqExec_x86_64_Ubuntu
+      - IN: execTemplates_repo
+      - IN: shipbits_ecr_v2_cli
+        scopes:
+          - ecr
+      - IN: ship_dh_cli
+      - IN: microbase_img
+      - TASK:
+        - script: ./IN/reqProc_repo/gitRepo/build.sh drydock x86_64 Ubuntu
+
```
Comment (Bharath92) [inline shippable.jobs.yml]: Should we have a `reqProc_img` as an OUT here?

Comment (shrivara) [reply shippable.jobs.yml]: Good point. I think we'll need it at some point, but for now its not required.

**Pull request merged**